torch/_torch_docs.py

```diff
@@ -3587,6 +3587,54 @@ def merge_dicts(*dicts):
     array([-1,  2,  3])
 """)
 
+add_docstr(torch.frombuffer,
+           r"""
+frombuffer(buffer, dtype=None, count=-1, offset=0, *, requires_grad=False) -> Tensor
+
+Create a 1-dimensional :class:`Tensor` from an object that implements
+the Python buffer protocol.
+
+Interpret :attr:`buffer` as a 1-dimensional buffer of type :attr:`dtype`
```

Comment:
Nit: "Create" -> "Creates" on line 3594 and "Interpret" -> "Interprets" here. 

What happens if the dtype is not specified? Maybe this paragraph should add a sentence at the end which explains the default behavior: "By default, the returned tensor has dtype X and contains all the values in the buffer."

Reply:
Actually, follow-up question: is that the default? Is the buffer always interpreted as a byte buffer, or are the values in the buffer interpreted as values? Like if I have a Python array [1, 2, 3] and I interpret that in float, do I get back [1., 2., 3.]?

Reply:
It looks like NumPy always interprets the array as bytes and just writes to the bytes directly

Reply:
> What happens if the dtype is not specified? Maybe this paragraph should add a sentence at the end which explains the default behavior: "By default, the returned tensor has dtype X and contains all the values in the buffer."

When `dtype` is not specified, the default scalar type is used. I will add it to the docs.

> is that the default? Is the buffer always interpreted as a byte buffer, or are the values in the buffer interpreted as values?

This implementation behaves the same as NumPy. So we use the bytes in the buffer as they are, but interpret them as the specified `dtype`. Maybe I should change the wording there, since it's not completely clear. What do you think about:

_"Skip the first `offset` bytes in `buffer`, and interpret the rest of them as a 1-dimensional tensor of type `dtype` with `count` elements"_